src/main/java/juja/microservices/keepers/service/KeepersService.java

```diff
@@ -67,19 +67,22 @@ public String addKeeper(KeeperRequest keeperRequest){
         logger.debug("Service.getActiveKeepers after in, without any parameters.");
         Map<String, List<String>> keepersWithDirections = new HashMap<>();
         List<Keeper> keepers = keepersRepository.getActiveKeepers();
+        logger.info("Get List<Keeper> : {}", keepers);
         for (Keeper keeper : keepers) {
             List<String> directions = new ArrayList<>();
-            if(keepersWithDirections.containsKey(keeper.getUuid())) {
-                directions = keepersWithDirections.get(keeper.getUuid());
-                directions.add(keeper.getDirection());
-                keepersWithDirections.replace(keeper.getUuid(),directions);
+            String keeperUuid = keeper.getUuid();
+            String keeperDirection = keeper.getDirection();
+            if(keepersWithDirections.containsKey(keeperUuid)) {
+                directions = keepersWithDirections.get(keeperUuid);
+                directions.add(keeperDirection);
+                keepersWithDirections.replace(keeperUuid,directions);
             }else{
-                directions.add(keeper.getDirection());
-                keepersWithDirections.put(keeper.getUuid(), directions);
+                directions.add(keeperDirection);
+                keepersWithDirections.put(keeperUuid, directions);
             }
         }
-        logger.info("Get List<Keeper> : {}, and create Map<String, List<String>> : {} from income list.", keepers,keepersWithDirections);
-        logger.debug("Service.getActiveKeepers before out, parameters: {}", keepersWithDirections);
+        logger.info("Create Map<String, List<String>> : {} from income list of Keepers.", keepersWithDirections);
```

Comment:
done : logger.debug("Create Map<String, ActiveKeeperDTO>. It has {} elements.", activeKeeperDTOMap.size());